test/gtest/aca_test_dhcp.cpp

```diff
@@ -0,0 +1,176 @@
+// Copyright 2019 The Alcor Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "gtest/gtest.h"
+#define private public
+#include "aca_dhcp_server.h"
+#include "aca_dhcp_programming_if.h"
+
+using namespace aca_dhcp_server;
+using namespace aca_dhcp_programming_if;
+
+TEST(dhcp_config_test_cases, add_dhcp_entry_valid)
```

Comment:
For each major category of UTs, can we add a few lines of comments?

Reply:
sure, that's easy, will add that before merging.